pkg/volume/volume_linux.go

```diff
@@ -73,17 +89,7 @@ func SetVolumeOwnership(mounter Mounter, fsGroup *int64) error {
 			klog.Errorf("Chown failed on %v: %v", path, err)
 		}
 
-		mask := rwMask
-		if mounter.GetAttributes().ReadOnly {
-			mask = roMask
-		}
-
-		if info.IsDir() {
-			mask |= os.ModeSetgid
-			mask |= execMask
-		}
-
-		err = os.Chmod(path, info.Mode()|mask)
+		err = os.Chmod(path, calculateMask(mounter, info.Mode(), info.IsDir()))
 		if err != nil {
 			klog.Errorf("Chmod failed on %v: %v", path, err)
 		}
```

Comment:
Does Errorf throw a panic? Otherwise are we dying here?

Reply:
just a spam log, not like fatal